jenkins/root-build.cmake

```
@@ -439,6 +443,14 @@ function(GET_ALL_SUPPORTED_MODULES_LINUX)
     )
   endif()
 
+  # Ubuntu 18.04 has a system python-numba package which is too old.
+  # All other distro versions (14, 16 and 20 up to now) don't have a
+  # distro package and we install via pip.
```

That's not totally correct IIRC:
```suggestion
  # Ubuntu 18.04 has a system python-numba package which is too old.
  # Older distro versions (14, 16) don't have a distro package and we install via pip.
  # Newer ones (20 and up) have sufficiently new distro versions.
```